Day1-Session2-Lecture/session2-dynamics-draft.ipynb

```diff
@@ -20,7 +19,6 @@
   },
   {
    "cell_type": "markdown",
-   "id": "impressed-dream",
```

Comment:
🤔 Your Jupyter is much more creative than mine -- in fact my cells don't have ids...